dpx/src/dpx_pdfdev.rs

```diff
@@ -629,32 +629,21 @@ unsafe fn text_mode() {
 }
 #[no_mangle]
 pub unsafe extern "C" fn graphics_mode() {
-    let mut current_block_3: u64;
     match motion_state {
-        MotionState::STRING_MODE => {
-            pdf_doc_add_page_content(if text_state.is_mb != 0 {
-                b">]TJ"
-            } else {
-                b")]TJ"
-            });
-            current_block_3 = 13064676843759196241;
-        }
-        MotionState::TEXT_MODE => {
-            current_block_3 = 13064676843759196241;
-        }
-        MotionState::GRAPHICS_MODE => {
-            current_block_3 = 11875828834189669668;
-        }
-    }
-    match current_block_3 {
-        13064676843759196241 =>
-        /* continue */
-        {
+        MotionState::GRAPHICS_MODE => {}
+        MotionState::STRING_MODE|MotionState::TEXT_MODE => {
+            if let MotionState::STRING_MODE = motion_state {
+                pdf_doc_add_page_content(if text_state.is_mb != 0 {
+                    b">]TJ"
+                } else {
+                    b")]TJ"
+                });
+            }
+            
             pdf_doc_add_page_content(b" ET"); /* op: ET */
             text_state.force_reset = 0i32;
             text_state.font_id = -1i32
```

Comment:
You can get is slightly cleaner with `MotionState::GRAPHICS_MODE => return,` but this is fine